src/ripple/test/jtx/Env.h

```diff
@@ -103,13 +111,20 @@ struct noripple_helper
 
 /** Designate accounts as no-ripple in Env::fund */
 template <class... Args>
-detail::noripple_helper<1 + sizeof...(Args)>
-noripple (Account const& account,
-    Args const&... args)
+detail::vararg_array_helper<Account, 1 + sizeof...(Args)>
+noripple (Account const& account, Args const&... args)
+{
+    return detail::vararg_array_helper<Account,
+        1 + sizeof...(Args)>(account, args...);
+}
+
+/** Activate features in the Env ctor */
+template <class... Args>
+detail::vararg_array_helper<uint256, 1 + sizeof...(Args)>
+features (uint256 const& key, Args const&... args)
 {
-    return detail::noripple_helper<
-        1 + sizeof...(Args)>(
-            account, args...);
+    return detail::vararg_array_helper<uint256,
+        1 + sizeof...(Args)>(key, args...);
```

Comment:
`return {{key, args...}};` works too (tested with the `std::array` modification posted earlier)